test/utils/tree-view/describeTreeView/describeTreeView.tsx

```diff
@@ -177,13 +187,14 @@ const innerDescribeTreeView = <TPlugin extends TreeViewAnyPluginSignature>(
         );
 
         const result = render(
-          <SimpleTreeView slots={slots} {...other}>
+          <SimpleTreeView slots={slots} apiRef={apiRef} {...other}>
```

Comment:
I don't want to add `!` on every single test when we know the ref will never be undefined